app/components/frame-viewer.cjsx

```diff
@@ -200,17 +197,18 @@ module.exports = React.createClass
 
   continuousZoom: (change) ->
     return if change == 0
-    @setState zooming: true
-
-    zoomNow = =>
-      @zoom(change)
-      clearTimeout @state.zoomingTimeoutId
-      @setState zoomingTimeoutId: setTimeout(zoomNow, 200)
-    
-    zoomNow()
+    @setState zooming: true, =>
+      zoomNow = =>
+        # if !@state.zooming, we don't want to continuously call setTimeout
+        return if !@state.zooming
```

Comment:
Not quite sure what this check does, since you set `state.zooming` to true before calling this function.